client/src/components/SearchReviews.jsx

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import {
+  Wrapper,
```

Comment:
I see that you're using Wrapper a lot. If they're all basically the same thing, but in different files, I recommend using the same one. If not, ok.

Reply:
Good call.  You're right.  I will take a look!
*edit: it's likely that if they aren't the same, then they should have different names, so that in cases where I *do* want to reuse them in different places, there's no confusion.  So I'll be revisiting!